test/OpenTelemetry.Tests/Impl/Trace/SpanTest.cs

```diff
@@ -699,7 +699,7 @@ public void Dispose()
         private void AssertApproxSameTimestamp(DateTime one, DateTime two)
         {
             var timeShift = Math.Abs((one - two).TotalMilliseconds);
-            Assert.InRange(timeShift, double.Epsilon, 10);
+            Assert.InRange(timeShift, double.Epsilon, 20);
```

Comment:
@lmolkova This ok? I'm getting random failures from the build checks that don't happen locally.

Reply:
that's ok. Thanks for fixing it. I was thinking about a fancier solution (passing timestamp explicitly rather than compare nondeterministically), but current API does not allow it